src/reanimated2/animation/util.ts

```diff
@@ -38,7 +38,7 @@ const IS_REDUCED_MOTION = isReducedMotion();
 
 if (__DEV__ && IS_REDUCED_MOTION) {
   console.warn(
-    `[Reanimated] Reduced motion setting is enabled on this device. Some animations will be disabled by default. You can override the behavior for individual animations, see https://docs.swmansion.com/react-native-reanimated/docs/guides/accessibility.`
+    `[Reanimated] Reduced motion setting is enabled on this device. This warning is visible only in the development mode. Some animations will be disabled by default. You can override the behavior for individual animations, see https://docs.swmansion.com/react-native-reanimated/docs/guides/accessibility.`
```

Comment:
Maybe let's link to troubleshooting section instead?

Reply:
makes sense